nav2_lifecycle_manager/src/lifecycle_manager.cpp

```diff
@@ -145,6 +147,18 @@ LifecycleManager::isActiveCallback(
   response->success = system_active_;
 }
 
+void
+LifecycleManager::CreateActiveDiagnostic(diagnostic_updater::DiagnosticStatusWrapper & stat)
+{
+
+  if (system_active_) {
+    stat.summary(diagnostic_msgs::msg::DiagnosticStatus::OK, "Nav2 is active");
+  } else {
+    stat.summary(diagnostic_msgs::msg::DiagnosticStatus::ERROR, "Nav2 is inactive");
```

Comment:
I'm not sure `Error` is the right option, but it is true that there is no `inactive` or similar functionality in `DiagnosticStatus`